library/Requests.php

```diff
@@ -63,6 +63,13 @@ class Requests {
 	const PATCH = 'PATCH';
 
 	/**
+	 * Default size of buffer chunk size to read streams
+	 *
+	 * @var integer
+	 */
+	const CHUNK = 1160;
```

Comment:
We should probably call this `BUFFER_SIZE` or similar; chunk might get confusing when dealing with HTTP/1.1 chunked responses.